fineract-provider/src/main/java/org/apache/fineract/infrastructure/gcm/domain/MulticastResult.java

```diff
@@ -83,64 +92,14 @@ private MulticastResult(Builder builder) {
         retryMulticastIds = Collections.unmodifiableList(tmpList);
     }
 
-    /**
-     * Gets the multicast id.
-     */
-    public long getMulticastId() {
-        return multicastId;
-    }
-
-    /**
-     * Gets the number of successful messages.
-     */
-    public int getSuccess() {
-        return success;
-    }
-
-    /**
-     * Gets the total number of messages sent, regardless of the status.
-     */
-    public int getTotal() {
-        return success + failure;
-    }
-
-    /**
-     * Gets the number of failed messages.
-     */
-    public int getFailure() {
-        return failure;
-    }
-
-    /**
-     * Gets the number of successful messages that also returned a canonical registration id.
-     */
-    public int getCanonicalIds() {
-        return canonicalIds;
-    }
-
-    /**
-     * Gets the results of each individual message, which is immutable.
-     */
-    public List<Result> getResults() {
-        return results;
-    }
-
-    /**
-     * Gets additional ids if more than one multicast message was sent.
-     */
-    public List<Long> getRetryMulticastIds() {
-        return retryMulticastIds;
-    }
-
     @Override
     public String toString() {
```

Comment:
You can replace the toString with a @ToString lombok annotation.

Reply:
@toString is already included in @Data annotation